images/base/cloudbuild.yaml

```diff
@@ -3,5 +3,5 @@ options:
   substitution_option: ALLOW_LOOSE
 steps:
 - name: gcr.io/k8s-testimages/krte:latest-master
-  env:
-  entrypoint: make -C images/base push
+  entrypoint: make
+  args: ['-C' 'images/base' 'push']
```

Comment:
don't you need a comma?

Reply:
it seems you do https://cloud.google.com/build/docs/configuring-builds/create-basic-configuration

Reply:
yep